README.md

```diff
@@ -55,6 +55,26 @@ This is the place to:
 Note: there is no need to repeat everything that is already well-documented in
 the code.
 
+If you would like to include math equations to explain parts of your code or repo,
+it is recommended to use an external renderer and insert prepared svg files here.
```

Comment:
We've also used PNG images. Do you see a quality problem with those or another reason to require SVG? Perhaps we can say image files and note that SVG is preferred but not mandatory?

Reply:
PNG files could be ok, but it seems easy enough to me that we pick a standard for uniformity. SVG will generally always be higher quality and more flexible - GIF and PNG etc. will have pixelation issues at different resolutions.

Reply:
I was thinking of a use case where a contributor is using typical Windows tools to snapshot an equation from a PDF, for example. These usually have PNG, JPEG, etc but not SVG.
I agree that a high-res image is desirable but just worry that if creating an SVG is an obstacle to some users, they might forgo an equation, and I'd rather a lower quality image than no equation. So far I haven't seen low quality images with PNG. We would comment on those in a PR. (I typically view these in GitHub so maybe it's more problematic in the IDE.)

Reply:
That's a good point. I think the main quality issues I've seen are with GIF rendered equations, but in general I think uniform SVG would be a good look. A middle ground is to say in the any image file, and to have someone on our side take responsibility for translating anything that gets uploaded which isn't SVG? Using the url renderer it's as simple as typing out the Latex.